### PR TITLE
Fixed for Swift 4.0

### PR DESCRIPTION
**Pull request by drewmccormack**

### CODE REVIEW

File: Sources/SwiftHEXColors.swift [hunk 1]
```diff
@@ -71,7 +71,7 @@ public extension SWColor {
 
 		// Check for hash and remove the hash
 		if hex.hasPrefix("#") {
-			hex = hex.substring(from: hex.index(hex.startIndex, offsetBy: 1))
+			hex = String(hex[hex.index(hex.startIndex, offsetBy: 1)...])
```
Comment (alexandergriekspoor) [inline Sources/SwiftHEXColors.swift]: oh man, can't say I find the new syntax a lot more appealing :-( Maybe make it a two liner by doing:
```
let index = hex.index(hex.startIndex, offsetBy: 1)
hex = String(hex[index...])
```

Btw, we're sure this doesn't work?
`hex = String(hex[1...])`
If not I still find that something unbelievebly stupid that they didn't address that, given that they do allow you to now ask the count of a string (i.e. it should know how to convert things to integers)

**Review by alexandergriekspoor — APPROVED** [review]
One comment to look at

File: Sources/SwiftHEXColors.swift (hunk 1 above)
Comment (drewmccormack) [reply Sources/SwiftHEXColors.swift]: That was first thing I tried. Doesn't work. 
Have spit out the index calculation.

Comment (alexandergriekspoor) [reply Sources/SwiftHEXColors.swift]: Yeah was afraid of that, really a missed opportunity from Apple to really fix that ugly range API for strings. I mean if you now support `aString.count` then you basically have decided what "view" you're going to use to return an int, why then not also support direct int based ranges that use the same view ?!

**Pull request merged**